packages/flutter/lib/src/widgets/notification_listener.dart

```diff
@@ -58,15 +58,36 @@ abstract class Notification {
   /// Subclasses can override this to apply additional filtering or to update
   /// the notification as it is bubbled (for example, increasing a `depth` field
   /// for each ancestor of a particular type).
+  ///
+  /// This functionality was deprecated after v1.11.0 because of poor performance.
+  /// If you have a Notification subtype that overrides this method, you can instead
+  /// manually implement the old-style notification dispatch with the following
+  /// snippet:
+  ///
+  /// ```dart
+  ///  void dispatchNotificationSlowly<T extends Notification>(T notification, BuildContext context) {
+  ///    context.visitAncestorElements((Element element) {
+  ///       // Perform visit ancestor logic.
+  ///       notification.myAncestorLogic(element);
```

Comment:
It's not super clear what this is. Maybe say explicitly that this is were the custom filtering/update logic (mentioned in one of the paragraphs above) would go that prior to the deprecation went into `visitAncestor`?